docs/framework/migration-guide/versions-and-dependencies.md

```diff
@@ -24,6 +24,7 @@ Each version of the .NET Framework contains the common language runtime (CLR), t
   
 |.NET Framework version|CLR version|Included in<br /> Visual Studio<br/>version|✓ Included in<br />+ Can be installed on<br />Windows|✓ Included in<br />+ Can be installed on<br />Windows Server|To determine installed .NET version|  
 |----------------------------|-----------------|--------------|---------------------------------------|----------------------------------------------------|-----------------------------------------------------------|-----------------------------------------| 
+|4.8<br/><br/>[New features](../whats-new/index.md#whats-new-in-the-net-framework-48)<br/><br/>[New in accessibility](../whats-new/whats-new-in-accessibility.md#whats-new-in-accessibility-in-the-net-framework-48)|4| | ✓ 10 May 2019 Update<br/><br/> + 10 October 2018 Update (Version 1809) <br/><br/> ✓ 10 April 2018 Update (Version 1803) <br/><br/> + 10 Fall Creators Update (Version 1709) <br/> <br/> + 10 Creators Update (Version 1703) <br/> + 10 Anniversary Update (Version 1607) <br/> + 8.1 <br/> +7 | + Windows Server 2019<br/> + Windows Server, version 1809 <br/> + Windows Server, version 1803 <br/> + 2016 <br/> + 2012 R2 <br/> + 2012 <br/> + 2008 R2 SP1 |Use `Release` DWORD:<br/><br/> - 528040 (Windows 10 May 2019 Update) <br/> - 528049 (all other OS versions) <br/><br/> (see [instructions](../../../docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md))|
```

Comment:
As discussed offline, the + and ✓ looked inconsistently and spacing is different for each version.

You can also simplify one of the links

```suggestion
|4.8<br/><br/>[New features](../whats-new/index.md#whats-new-in-the-net-framework-48)<br/><br/>[New in accessibility](../whats-new/whats-new-in-accessibility.md#whats-new-in-accessibility-in-the-net-framework-48)|4| | ✓ 10 May 2019 Update<br/><br/> + 10 October 2018 Update (Version 1809) <br/><br/> ✓ 10 April 2018 Update (Version 1803) <br/><br/> + 10 Fall Creators Update (Version 1709) <br/> <br/> + 10 Creators Update (Version 1703) <br/> + 10 Anniversary Update (Version 1607) <br/> + 8.1 <br/> +7 | + Windows Server 2019<br/> + Windows Server, version 1809 <br/> + Windows Server, version 1803 <br/> + 2016 <br/> + 2012 R2 <br/> + 2012 <br/> + 2008 R2 SP1 |Use `Release` DWORD:<br/><br/> - 528040 (Windows 10 May 2019 Update) <br/> - 528049 (all other OS versions) <br/><br/> (see [instructions](how-to-determine-which-versions-are-installed.md))|
```